packages/manager/modules/server-sidebar/src/controller.js

```diff
@@ -170,66 +180,75 @@ export default class OvhManagerServerSidebarController {
     return link;
   }
 
+  buildOrderMenuItems(items, ovhSubsidiary) {
+    return map(items, (orderItemConfig) => {
+      if (
+        !has(orderItemConfig, 'featureType') ||
+        this.CucFeatureAvailabilityService.hasFeature(
+          orderItemConfig.featureType,
+          'sidebarOrder',
+          ovhSubsidiary,
+        )
+      ) {
+        const isExternal = !includes(orderItemConfig.app, this.universe);
+
+        let link = null;
+        if (
+          (isExternal || !has(orderItemConfig, 'state')) &&
+          (has(orderItemConfig, 'linkId') || has(orderItemConfig, 'linkPart'))
+        ) {
+          link = this.buildUrl(orderItemConfig, ovhSubsidiary);
+        }
+
+        let subActions = [];
+        if (has(orderItemConfig, 'children')) {
+          subActions = this.buildOrderMenuItems(
+            orderItemConfig.children,
+            ovhSubsidiary,
+          );
+        }
+
+        if (!isExternal || link || subActions.length > 0) {
```

Comment:
looks weird that isExternal is used as a condition here since it's also used inside of the "if" block
are you sure the isExternal condition is needed ?

Reply:
Here we are building an order item, if the link is not external (it's an internal app state) or if we have a link, or if we have subActions
As we are using `||` we can enter in this block if `isExternal` is true and `link` is defined, so we are using the same block for 3 independants conditions. That's why we use isExternal inside.